kalite/distributed/features/steps/superuser_create.py

```diff
@@ -5,6 +5,10 @@
 
 modal_container = "superusercreate-container"
 
+@given("I go to homepage")
```

Comment:
Have to agree with @MCGallaspy, it's important not to create two similar-worded givens, they will create confusion for everyone :)

"there is superuser" should perhaps be called "there exists a superuser account".